ex1.py

```diff
@@ -0,0 +1,13 @@
+print "Hello World!"
+print "Hello Again"
+print "I like typing this."
+print "This is fun"
+print "Yay! Printing."
+print "I'd much rather you 'not'."
+print 'I "said" do not touch this.'
+
+# Understands Following things in Ex.1
+# How to create .py file, execute the program, the python file on console.
```

Comment:
To execute the python file on terminal we need to write: pyhton file_name 
for eg: python ex1.py